site-src/v1alpha2/guides/simple-gateway.md

```diff
@@ -21,21 +18,19 @@ HTTP traffic on port 80. This particular GatewayClass automatically assigns an
 IP address which will be shown in the `Gateway.status` after it has been
 deployed. 
 
-Gateways bind Routes to themselves via label selection (similar to how Services
-label select across Pod labels). In this example, the `prod-web` Gateway will
-bind any HTTPRoute resources which have the `gateway: prod-web-gw` label. The
-label can be any arbitrary label, but using one that identifies the name or
-capabilities of the Gateway is useful to Route owners and makes the relationship
-more explicit. More complex bi-directional matching and permissions are possible
-and explained in other guides.
+An `HTTPRoute` resource must specify the Gateways it attaches to use
+`ParentRefs`. Referencing the `Gateway` will allow the `HTTPRoute` to
+receive traffic from the parent `Gateway`. `BackendRefs` define the backends
```

Comment:
Maybe something like this?
```suggestion
Route resources specify the Gateways they want to attach to using `ParentRefs`. As long as 
the Gateway allows this attachment (by default Routes from the same namespace are trusted),
this will allow the Route to receive traffic from the parent Gateway. `BackendRefs` define the backends
```